src/PackageResolver.php

```diff
@@ -20,9 +20,19 @@ class PackageResolver
 {
     private const SYMFONY_VERSIONS = ['lts', 'previous', 'stable', 'next'];
 
+    /**
+     * @var array
+     */
     private static $aliases;
+
+    /**
+     * @var array
```

Comment:
I don't like the inlined version, I have not executed php-cs-fixer on the package itself (not sure it's fully compatible with PHP7.1 yet, but moreover, I was lazy 😛), but I'm not sure it would allow inlined comments 🤔